CMakeLists.txt

```diff
@@ -36,29 +39,31 @@ target_link_libraries(LibEugeneCore
         CONAN_PKG::fmt
         CONAN_PKG::ms-gsl
         CONAN_PKG::outcome
+        CONAN_PKG::catch2
```

Comment:
Please get it back to normal. This change just expands the diff without any actual value. 😃

Reply:
Sure